ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Documentation       Post install test cases that verify OCP KServe resources and objects
+Library             OpenShiftLibrary
+
+
+*** Variables ***
+${TRUSTYAI_NS}=    ${APPLICATIONS_NAMESPACE}
+
+
+*** Test Cases ***
+Verify TrustyAI Operator Installation
+    [Documentation]    Verifies that the TrustyAI operator has been
+    ...    deployed in the ${APPLICATIONS_NAMESPACE} namespace in ODS
+    [Tags]    Smoke
+    ...       Tier1    ODS-2481
+    ...       OpenDataHub
```

Comment:
OpenDataHub tag no longer needed, as we expect all tests in Smoke to be compatible.  We have tag `ExcludeOnODH` for tests that should be excluded in ODH